src/test/java/org/apache/commons/dbutils/MockResultSet.java

```diff
@@ -21,10 +21,7 @@
 import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Don't change to start imports.

Reply:
Sure @garydgregory , 

I'm currently addressing the issue, and I appreciate your prompt review. 

Thank you.

Reply:
and please rebase on master ;-) TY!